docs/platform/webcontainers/browser-config.md

```diff
@@ -31,7 +31,7 @@ https://[*.]stackblitz.io
 https://[*.]webcontainer.io
 ```
 
-For instance, in Chrome, go to [chrome://settings/cookies](chrome://settings/cookies) and add those exceptions in the “Sites that can always use cookies” section.
+For instance, in Chrome, go to `chrome://settings/cookies` and add those exceptions in the “Sites that can always use cookies” section.
```

Comment:
Both Chrome and Firefox do not allow navigating from a web page to a URL using their internal scheme (`chrome://` in Chrome, `about:` in Firefox).

I think this used to work, which is why I made this a link for users's convenience, but it's probably been blocked for security reasons.